specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/expressRouteCircuit.json

```diff
@@ -1571,6 +1576,15 @@
       },
       "description": "Contains stats associated with the peering."
     },
+    "RouteFilterReference": {
+      "properties": {
```

Comment:
Please update the examples if any for these changes

Reply:
The examples would not need an update since the data returned was always an id.

Reply:
updated